### DISCUSSION TITLE
Size of latent space

### DISCUSSION THREAD

**Discussion by mo-seph** [acids-ircam/RAVE#241] [General]
I can never guess how large the latent space is likely to be for a given config, and it seems to change in the middle of runs - I've had some where it's 4 signals after some iterations, goes up to 32 in the middle of training, and eventually comes out at 16. Is that normal? Is there any way to ask for a particular size?

**Comment by josefhaeusel** [comment]
![image](https://github.com/acids-ircam/RAVE/assets/107403830/8e7a3f6f-8dba-47d5-bff0-7efd9a977012)

Check out the Appendix to their RAVE paper. Apparently, it's 128 dimensions. I can't explain, why it changes for you...
